tweak Makefile such that $CXX and $CXXFLAGS are picked up from the environment
B. Hajgato (Free University Brussels, VUB), Kenneth Hoste (HPC-UGent)
--- kraken2-2.0.6-beta/src/Makefile.orig	2018-06-26 21:19:26.000000000 +0200
+++ kraken2-2.0.6-beta/src/Makefile	2018-07-09 14:00:54.566891002 +0200
@@ -1,5 +1,5 @@
-CXX = g++
-CXXFLAGS = -fopenmp -Wall -std=c++11 -O3
+CXX ?= g++
+CXXFLAGS ?= -fopenmp -Wall -std=c++11 -O3
 CXXFLAGS += -DLINEAR_PROBING
 
 .PHONY: all clean install
